source/load-test-with-locust.rst

```diff
@@ -0,0 +1,479 @@
+Load test using Locust
```

Comment:
Maybe you can put a more generic title like Using load test to measure the environment supports, our something better...

Reply:
By mentioning load test, doesn't it already mean that we are measuring the environment support?

Reply:
I think I might not mention "locust" in the title, but something more meaningful for someone who doesn't know "locust". And also why this page could be interesting for anyone who is going to read.

Reply:
Fair enough!